vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -274,13 +279,24 @@ func (s *GenericAPIServer) PrepareRun() preparedGenericAPIServer {
 // Run spawns the secure http server. It only returns if stopCh is closed
 // or the secure port cannot be listened on initially.
 func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
+	eventRef := &corev1.ObjectReference{
```

Comment:
isn't there helper in client-go somewhere that does this?

Reply:
there are helpers taking a `runtime.Object` (which we don't have here)